submissions/ArthurGorbenko/html.css.practice/index.html

```diff
@@ -0,0 +1,175 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+
+<title>Html/css practice</title>    
+
+<link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+    <main>      
+        <div class="container">
+                <div class="mail_button"><a href="" class="link">Gmail</a></div>
+                <div class="Images"><a href="" class="link">Images</a></div>
+                <input type="checkbox" id="trig" name="trigger">
+                <label for="trig" class="button" id="menu_btn">
+                         <svg viewBox="0 0 24 24" class="menu_btn">
+                            <path d="M 6 8 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 12 20 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 6 20 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 6 14 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 12 14 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 16 6 c 0 1.1 0.9 2 2 2 s 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 Z M 12 8 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 18 14 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z M 18 20 c 1.1 0 2 -0.9 2 -2 s -0.9 -2 -2 -2 s -2 0.9 -2 2 s 0.9 2 2 2 Z">
+                            </path>
+                         </svg>
+                    </label>
+                <div class="content">
+                <input type="checkbox" id="more">   
+                <div class="list_2_container">
+                    <ul class="list_2">
+                      <li class="card">
+                          <a href="" class="link">
+                           <img src="account.png" class="icon">
```

Comment:
In case if the project used many small icons, that is recommended to use CSS Sprites instead of many single images. That reduces the number of http requests to the server and other positive influences.

Reply:
I guess, something like [this](https://github.com/kottans/frontend-2019-p2p/pull/67/commits/a9cd8a9edcddefb526cb8c93c6156819a45dd284).